controller/controller_server/csi_controller_server.py

```diff
@@ -581,8 +582,9 @@ def ControllerExpandVolume(self, request, context):
 
                 if required_bytes > max_size:
                     message = messages.SizeOutOfRangeError_message.format(required_bytes, max_size)
-                    context.set_details(message)
+                    logger.exception(message)
```

Comment:
did you mean [`logger.error`](https://docs.python.org/3/library/logging.html#logging.error)?

Reply:
yes. it looks right

Reply:
done